tests/dummy/app/router.js

```diff
@@ -1,7 +1,8 @@
-import Ember from 'ember'
 import config from './config/environment'
+import Ember from 'ember'
+const {Router: EmberRouter} = Ember
 
-var Router = Ember.Router.extend({
+var Router = EmberRouter.extend({
```

Comment:
I would prefer that we not do this since we only have one Router library right now.